qemu/tests/cfg/blockdev_commit_server_down.cfg

```diff
@@ -13,23 +13,23 @@
 
     # The following is specified for a local data image 'data',
     # which will be exported by qemu-nbd
-    local_image_tag = data
-    image_size_data = 2G
-    image_name_data = data
-    nbd_port_data = 10810
+    local_image_tag = nbddata
+    image_size_nbddata = 2G
+    image_name_nbddata = nbddata
+    nbd_port_nbddata = 10810
 
     # The following is specified for nbd image 'nbddata',
     # i.e. the exported 'data', used as a data disk in VM
```

Comment:
@aliang123 Please also update the comments. Thanks

Reply:
Have updated, thanks.